### PR TITLE
Build: Tests using enzyme

### PR DESCRIPTION
**Pull request by vinteo**

### CODE REVIEW

File: test/components/adslotUi/UserListPickerComponentTest.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
+
```
Comment (camjc) [inline test/components/adslotUi/UserListPickerComponentTest.js]: bonus

File: test/components/adslotUi/TreePickerPureComponentTest.js [hunk 2]
```diff
@@ -12,65 +15,39 @@ describe('TreePickerPureComponent', () => {
     rootTypes,
   } = TreePickerMocks;
 
-  const indices = immutable({
-    leftPane: 0,
-    rightPane: 1,
-  });
+  it('should render with defaults', () => {
+    const component = shallow(<TreePickerPureComponent changeRootType={_.noop} />);
```
Comment (camjc) [inline test/components/adslotUi/TreePickerPureComponentTest.js]: :+1: noooooop

File: test/components/adslotUi/ListPickerPureComponentTest.js [hunk 3]
```diff
@@ -20,155 +20,122 @@ describe('ListPickerPureComponent', () => {
...
-      allowMultiSelection: false,
-      items: users,
-      selectedItems,
+    const props = { allowMultiSelection: false, items: users, selectedItems };
+    const component = shallow(<ListPickerPureComponent {...props} />);
+    expect(component.prop('className')).to.equal('listpickerpure-component');
+
+    const gridElement = component.find(Grid);
+    const gridRowElements = gridElement.find(GridRow);
+    gridRowElements.forEach((gridRowElement, index) => {
```
Comment (camjc) [inline test/components/adslotUi/ListPickerPureComponentTest.js]: :+1: Good move.

File: src/components/Main.js [hunk 4]
```diff
@@ -228,7 +228,7 @@ class AppComponent extends React.Component {
         </div>
 
         <h1>Modal</h1>
-        <Button className="btn-inverse" onClick={this.toggleSimpleModal}>
+        <Button id="button-modal" className="btn-inverse" onClick={this.toggleSimpleModal}>
```
Comment (camjc) [inline src/components/Main.js]: can we do these as data test selectors? I don't want to implicitly encourage `id`s.

**Comment by vinteo** [comment]
@camjc its because I reinstalled all my modules and there are patch updates for `react-icheck` and `react-bootstrap`

**Comment by camjc** [comment]
Merging

**Pull request merged**